MAX17048/MAX17048.pro

```diff
@@ -0,0 +1,33 @@
+update=22/05/2015 07:44:53
```

Comment:
Change nickname of star-common-lib library to star-common-lib instead of star-common-lib